modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -129,6 +134,25 @@ export default {
   methods: {
     emitEvent: function (name) {
       apos.bus.$emit('admin-menu-click', name);
+    },
+    filterEmptySchema(menu) {
```

Comment:
The global doc has no editable schema by default. It's unlikely but possible for other modules to be in the same situation. There could be some edge case where someone wants a manager for a type that you can't actually edit... maybe. But is there harm in not supporting that by default?

Reply:
I would go so far as to say it will not happen, but not so far as to say you're wrong for preparing for it.

Reply:
I'm going to go about this differently instead.